CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
 # Changelog for Elixir v1.9
 
+## Releases
+
+The main feature in Elixir v1.9 is the addition of releases. A release is a self-contained directory with your application code, all of its dependencies, plus the whole Erlang VM and runtime. Once a release is assembled, it can be packaged and deployed to a target, as long as the target runs on the same operating system (OS) distribution and version as the machine running the `mix release` command.
+
+You can start a new project and assemble a release for it in 3 easy steps:
+
+    $ mix new my_app
+    $ cd my_app
+    $ MIX_ENV=prod mix release
+
+A release will be assembled in `_build/prod/rel/my_app`. Inside the release, there will be a `bin/my_app` file which is the entry point to your system. It supports multiple commands, such as:
+
+  * `bin/my_app start`, `bin/my_app start_iex`, `bin/my_app restart`, and `bin/my_app stop` - for general management of the release
+
+  * `bin/my_app rpc COMMAND` and `bin/my_app remote` - for running commands on the running system or to connect to the running system
+
+  * `bin/my_app eval COMMAND` - to start a fresh system that runs a single command and then shuts down
+
+  * `bin/my_app daemon` and `bin/my_app daemon_iex` - to start the system as a daemon on Unix-like systems
+
+  * `bin/my_app install` - to install the system as a service on Windows machines
+
+### Why releases?
+
+Releases allow developers to precompile and package all of their code and the runtime into a single unit. The benefits of releases are:
+
+  * Code preloading. The VM has two mechanisms for loading code: interactive and embedded. By default, it runs in the interactive mode which dynamically loads modules when they are used for the first time. The first time your application calls `Enum.map/2`, the VM will find the `Enum` module and load it. There’s a downside. When you start a new server in production, it may need to load many other modules, causing the first requests to have an unusual spike in response time. Releases run in embedded mode, which loads all available modules upfront, guaranteeing your system is ready to handle requests after booting.
```

Comment:
```suggestion
  * Code preloading. The VM has two mechanisms for loading code: interactive and embedded. By default, it runs in the interactive mode which dynamically loads modules when they are used for the first time. The first time your application calls `Enum.map/2`, the VM will find the `Enum` module and load it. There’s a downside; when you start a new server in production, it may need to load many other modules, causing the first requests to have an unusual spike in response time. Releases run in embedded mode, which loads all available modules upfront, guaranteeing your system is ready to handle requests after booting.
```

Reply:
We don't use `;` in Elixir documentation to keep the rules simple.